package/src/components/Channel/__tests__/isMessageEqual.test.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```

Comment:
Can we convert this test to TS? We are making it a standard as we go

Reply:
Happy to give it a shot